### PR TITLE
Content group simple - content model

### PR DESCRIPTION
**Pull request by mfearer94**
This is a new variation of content model that i'm trying out for the content group simple. The reason it is different is because there is no page for content item which is used in this component. So either a recommend this content model layout or some variation of it or we could add a new page for content item. We currently have one for content item-horizontal, but not just content item. Please give thoughts.

### Changelog

**New**

- Content model
- Anchor link

### CODE REVIEW

**Comment by JennySanchez** [comment]
Is there an extra space after text link CTA (text) and CTA (card)? Can we double check, @mfearer94? 😬

**Comment by mfearer94** [comment]
@JennySanchez Just double checked and there is no space afterwards. It must be how the underline looks under ")"

File: src/pages/components/content-group-simple.mdx [hunk 1]
```diff
@@ -23,9 +24,24 @@ Content group &mdash; simple is ideal for grouping content within sections with
...
+
+| Field | Sub-component field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ---------- | ---------- |
+| Headline | | Short copy | Yes | 1 | XS (65 / 85)|
+| Paragraph | | Long copy | No | 1 | XXL (350 / 450)|
+| [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) or [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) | | Component | No | 1 | - |
+| Content item | | Component | Yes | 2-6 | - |
+| | Headline | Short copy | Yes | 1 | XS (65 / 85) |
+| | Paragraph | Long copy | Yes | 1 | XXXXL (1600 / 2400) |
+| | [CTA (text)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | No | 1 | - |
```
Comment (photodow) [inline src/pages/components/content-group-simple.mdx:37]: I kind of like the addition of sub component fields here. I think image or video is allowed at the content item level too if I'm not mistaken.

File: src/pages/components/content-group-simple.mdx [hunk 2]
```diff
@@ -23,9 +24,24 @@ Content group &mdash; simple is ideal for grouping content within sections with
 
 ![Default](../../images/pattern/content-group-simple/default.jpg)
 
+## Content guidance
+
+| Field | Sub-component field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ---------- | ---------- |
+| Headline | | Short copy | Yes | 1 | XS (65 / 85)|
+| Paragraph | | Long copy | No | 1 | XXL (350 / 450)|
```
Comment (photodow) [inline src/pages/components/content-group-simple.mdx:32]: Just calling out content block for what it's worth.

File: src/pages/components/content-group-simple.mdx (hunk 1 above)
Comment (mfearer94) [reply src/pages/components/content-group-simple.mdx:37]: @photodow Would we want them to do that with this component?

Comment (mfearer94) [reply src/pages/components/content-group-simple.mdx:37]: @photodow Need your thoughts on this one.

**Comment by JennySanchez** [comment]
@mfearer94 Since our example above the table does not have the image or video under the Content items, let's move forward without it for now.

**Comment by mfearer94** [comment]
@JennySanchez This is ready for your final review.

**Review by JennySanchez — APPROVED** [review]
Thank you, MIchael.

**Pull request merged**